source/common/grpc/async_client_impl.cc

```diff
@@ -22,6 +22,9 @@ AsyncClientImpl::AsyncClientImpl(Upstream::ClusterManager& cm,
 
 AsyncClientImpl::~AsyncClientImpl() {
   while (!active_streams_.empty()) {
+    FANCY_LOG(info, "destroying active stream {} of async client imp {}",
+              static_cast<const void*>(active_streams_.front().get()),
+              static_cast<const void*>(this));
```

Comment:
Would be possible to log something about the AsyncClientImpl involved other than pointers?  Pointers don't say much about what the AsyncClientImpl was used for.

Also, is the log level of info appropriate or should this be debug?

Reply:
Sorry, this log is the garbage when I debug code. Addresses make sense only for gdb :)

It should be completed removed. 
Or, let me if I can populate meaning info other than addresses

Reply:
Removing the log is also acceptable.

Reply:
removed